lib/cuffsert/rxcfclient.rb

```diff
@@ -4,13 +4,10 @@
 
 module CuffSert
   class RxCFClient
-    def initialize(
-        aws_cf = Aws::CloudFormation::Client.new(retry_limit: 8),
-        pause: 5,
-        max_items: 1000)
-      @cf = aws_cf
-      @max_items = max_items
-      @pause = pause
+    def initialize(cli_args, **options)
+      @cf = options[:aws_cf] || Aws::CloudFormation::Client.new(region: cli_args[:aws_region], retry_limit: 8)
```

Comment:
I assume that when `cli_args[:aws_region]` the region will default to the built in resolution (i.e. `AWS_REGION` or `AWS_DEFAULT_REGION`)?

Reply:
As it turns out, it does not but rather thinks it has received `nil` for a region. Fixed.